kfp_notebook/pipeline/_notebook_op.py

```diff
@@ -203,6 +212,15 @@ def __init__(self,
             self.container.add_env_variable(V1EnvVar(name='PYTHONPATH',
                                                      value=self.python_user_lib_path))
 
+        if self.cpu_request:
+            self.container.set_cpu_request(cpu=str(cpu_request))
+
+        if self.mem_request:
+            self.container.set_memory_request(memory=str(mem_request) + "G")
+
+        if self.gpu_request:
+            self.container.set_gpu_limit(gpu=str(gpu_request))
```

Comment:
Since there's only GPU _limit_ (not _request_), should the parameter name change to `gpu_limit` (at least in kfp-notebook)?  I could see folks wanting to set limits and/or there eventually being a way to _request_ GPU - which may then cause some confusion here.

Do we need to provide a means for specifying 'amd' over the default 'nvidia'?  This might be something we could address via an ENV since I would imagine most installations use one vendor for their GPUs, but that may be a little optimistic.

ref: [`set_gpu_limit()`](https://kubeflow-pipelines.readthedocs.io/en/latest/_modules/kfp/dsl/_container_op.html)

Reply:
done and done